db/migrate/20150531152732_generate_tokens_for_existing_users.rb

```diff
@@ -0,0 +1,8 @@
+class GenerateTokensForExistingUsers < ActiveRecord::Migration
+  def change
```

Comment:
Better way than this is creating custom rake task. Take a look [here](http://guides.rubyonrails.org/command_line.html#custom-rake-tasks). Data migrations should not be combined with schema migrations.